build/Model_Output.txt

```diff
@@ -0,0 +1,107 @@
+! ---------
```

Comment:
Can we move this to the `docs/howto` folder perhaps? We can expand on this later. I want to move the documentation to http://summa.readthedocs.org, where we can have version dependent documentation (Joe did the same for VIC: http://vic.readthedocs.org). I already claimed the site, but there is nothing there yet. 

If you go to http://vic.readthedocs.org/en/latest, then you see a button on the bottom right that allows you to select branch-specific documentation.